src/lib.rs

```diff
@@ -926,7 +932,11 @@ externs! {
 
         fn __wbindgen_string_new(ptr: *const u8, len: usize) -> u32;
         fn __wbindgen_number_new(f: f64) -> u32;
-        fn __wbindgen_bigint_new(ptr: *const u8, len: usize) -> u32;
+        fn __wbindgen_bigint_from_str(ptr: *const u8, len: usize) -> u32;
+        fn __wbindgen_bigint_from_i64(n: i64) -> u32;
+        fn __wbindgen_bigint_from_u64(n: u64) -> u32;
+        fn __wbindgen_bigint_from_i128(hi: i64, lo: u64) -> u32;
+        fn __wbindgen_bigint_from_u128(hi: u64, lo: u64) -> u32;
```

Comment:
Why not pass i128/u128 directly instead of hi/lo and computing the value later?

Reply:
wasm-bindgen can't pass them natively yet, see #2822.

Reply:
Also, it would have to generate similar code even if it could, because Wasm JS BigInt integration currently allows only passing 64-bit values.